views/login-confirmation.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="{{ localeInfo.lang }}" dir="{{ localeInfo.direction }}">
+  <head>
+
+    <meta charset="utf-8">
+    {{ newrelic.getBrowserTimingHeader() | safe }}
+    <title>{{ gettext("Publish Your Remix") }}</title>
+
+    <link rel="stylesheet" href="/bower/makerstrap/dist/makerstrap.min.css" />
+    <link rel="stylesheet" href="/bower/webmaker-login-ux/dist/css/webmakerLogin.css">
+    <link rel="stylesheet" href="../stylesheets/style.css">
+    <link rel="stylesheet" href="../stylesheets/dialog.css">
+
+    <style>
```

Comment:
silly styling for now - UX is being looked at separately (tracked on the teach repository)